flowsa/flowbyactivity.py

```diff
@@ -143,6 +144,8 @@ def main(**kwargs):
 
     # assign yaml parameters (common.py fxn)
     config = load_sourceconfig(kwargs['source'])
+    # update the local config with today's date
+    config['date_generated']= pd.to_datetime('today').strftime('%Y-%m-%d')
```

Comment:
@catherinebirney I recognize that no bib is made during the FBA generation process, but I figured best to update the date here locally too after its pulled in, in case in the future the bib gets generated automatically.

Reply:
Ah ok, good call